stock_request_picking_type/models/stock_request_order.py

```diff
@@ -9,30 +9,25 @@ class StockRequestOrder(models.Model):
 
     @api.model
     def _get_default_picking_type(self):
+        companies = self._context.get("allowed_company_ids", []).copy()
```

Comment:
@ps-tubtim Why having used this instead a search on companies ?

Reply:
Moreover, I'm wondering why the security rule on picking type is not sufficent ?

https://github.com/odoo/odoo/blob/13.0/addons/stock/security/stock_security.xml#L75